src/main.rs

```diff
@@ -7,13 +7,60 @@ use skia_safe::{Color, Data, IRect, Image, Point, RRect, Rect};
 
 use std::io::{self, Read, Write};
 
+use clap::Parser;
+
 fn new_canvas(width: i32, height: i32) -> Surface {
     let mut surface = Surface::new_raster_n32_premul((width, height)).expect("no surface!");
     surface.canvas().clear(Color::TRANSPARENT);
     surface
 }
 
+struct ShadowColor {
+    r: u8,
+    g: u8,
+    b: u8,
+    a: u8,
+}
+
+impl From<String> for ShadowColor {
+    fn from(value: String) -> Self {
+        let r: u8 = u8::from_str_radix(&value[2..4],  16).unwrap_or(0);
+        let g: u8 = u8::from_str_radix(&value[4..6],  16).unwrap_or(0);
+        let b: u8 = u8::from_str_radix(&value[6..8],  16).unwrap_or(0);
+        let a: u8 = u8::from_str_radix(&value[8..10], 16).unwrap_or(0);
+
+        Self { r, g, b, a }
+    }
+}
+
+impl Into<Color> for ShadowColor {
+    fn into(self) -> Color {
+        Color::from_argb(self.a, self.r, self.g, self.b)
+    }
+}
+
+#[derive(Parser, Debug)]
+#[command(name = "shadower")]
+#[command(about = "Adds shadow and padding to images", long_about = None)]
```

Comment:
could you pls also mention the border radius

Reply:
aight